lib/OpenQA/WebAPI.pm

```diff
@@ -216,9 +216,9 @@ sub startup {
 
     # Favicon
     $r->get('/favicon.ico' => sub { my $c = shift; $c->render_static('favicon.ico') });
+    $r->get('/index' => [format => ['html', 'json']])->to('main#index');
```

Comment:
I think it is not necessary to specify the formats explicitly: `$r->get('/index')->to('main#index');`